cmd/up.go

```diff
@@ -443,6 +443,8 @@ func (up *UpContext) runCommand() error {
 	}
 
 	log.Infof("starting remote command")
+	command := strings.Join(up.Dev.Command, "\" \"")
+	command = fmt.Sprintf("PATH=$PATH:/var/okteto/bin \"%s\"", command)
```

Comment:
I wonder if this is somethign we should instead do in the dev env container image

Reply:
All the point is to make them available and transparently in every dev image.
I have tried another approach that I like it better.
We run a custom exec command before the user start command to kill previouos command executions. We can add logic there to move the binaries from `/var/okteto/bin` to `/usr/local/bin` if they don't exist in `/usr/local/bin` (`cp -n`).